src/Components/Browser.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -353,6 +366,18 @@ export class BrowserRenderer {
     }
   }
 
+  private findClosestAncestorSelectElement(element: Element | null) {
+    while (element) {
+      if (element instanceof HTMLSelectElement) {
+        return element;
+      } else {
+        element = element.parentElement;
+      }
+    }
+
+    return null;
```

Comment:
Is this the right approach? Wouldn't this silently fail? Should we at least log that we didn't find a select element? Am I being overly cautious here?

Reply:
I think you are being overly cautious. A developer could choose to have an `<option>` on its own (not nested inside a `<select>`). It wouldn't do anything useful, but browsers tolerate it. I don't think we should be trying to create extra errors in these cases, so I'm happy with the behavior of just doing nothing with such `<option>` elements.